### PR TITLE
Add new MAAS takeover

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add new MAAS takeover

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/))
- Check the homepage takeover in s, m and l

## Issue / Card

Card: https://trello.com/c/mI6jelii
Doc: https://docs.google.com/document/d/1IbyGV-s-LVFzGZCeE7nSo4ynAfvqiZLmqfNE4swddLQ/edit
Designs: https://www.dropbox.com/home/00_web_team/_ubuntu.com/takeovers/MAAS%2005.17/Exportables

### CODE REVIEW

File: templates/takeovers/_maas_may_2017.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<section class="row row-hero strip no-border">
+    <div class="strip-inner-wrapper equal-height">
```
Comment (deadlight) [inline templates/takeovers/_maas_may_2017.html]: 2 space indents, please.

**Comment by grahambancroft** [comment]
[Maas takeover build](https://trello.com/c/mI6jelii/3268-maas-takeover-build)

**Comment by grahambancroft** [comment]
Demo here: http://www.ubuntu.com-maas-takeover.demo.haus/

File: templates/takeovers/_maas_may_2017.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        <div class="six-col equal-height__item equal-height__item equal-height__align-vertically">
+            <h1 class="six-col">Maximise hardware efficiency</h1>
+            <p class="six-col intro">Build your data centre with <abbr title="Metal as a Service">MAAS</abbr> 1.9 is here.</p>
```
Comment (anthonydillon) [inline templates/takeovers/_maas_may_2017.html]: Isn't it MAAS 2.2?

File: templates/takeovers/_maas_may_2017.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+            <div class="row-hero__image-for-small for-small"><img src="{{ ASSET_SERVER_URL }}3fb3f433-MAAS-takeover-illustration.svg" alt="JuJu service blocks illustration" /></div>
```
Comment (anthonydillon) [inline templates/takeovers/_maas_may_2017.html]: It's Juju not JuJu

Comment (anthonydillon) [inline templates/takeovers/_maas_may_2017.html]: Please update the alt tag

File: templates/takeovers/_maas_may_2017.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+            <p><a href="https://pages.ubuntu.com/eBook-MAAS.html?utm_source=ubuntu.com&amp;utm_medium=takeover&amp;utm_campaign=MAAS_eBook " onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Maas takeover - 04-01-2016', 'eventLabel' : 'Learn more link', 'eventValue' : undefined });" class="button--primary">Download the eBook</a></p>
```
Comment (anthonydillon) [inline templates/takeovers/_maas_may_2017.html]: Update the onClick value to something about an ebook.

File: templates/takeovers/_maas_may_2017.html (hunk 1 above)
Comment (grahambancroft) [inline templates/takeovers/_maas_may_2017.html]: Done

File: templates/takeovers/_maas_may_2017.html (hunk 2 above)
Comment (grahambancroft) [inline templates/takeovers/_maas_may_2017.html]: Fixed

File: templates/takeovers/_maas_may_2017.html (hunk 3 above)
Comment (grahambancroft) [inline templates/takeovers/_maas_may_2017.html]: Fixed

File: templates/takeovers/_maas_may_2017.html (hunk 4 above)
Comment (grahambancroft) [inline templates/takeovers/_maas_may_2017.html]: Done

**Comment by grahambancroft** [comment]
All changes made, demo updated

**Comment by joanasa89** [comment]
looks good to me 👍

File: templates/takeovers/_maas_may_2017.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<section class="row row-hero strip no-border">
+  <div class="strip-inner-wrapper equal-height">
+    <div class="six-col equal-height__item equal-height__item equal-height__align-vertically">
+      <h1>Maximise hardware efficiency</h1>
+      <p class="six-col intro">Build your data centre with <abbr title="Metal as a Service">MAAS</abbr>.</p>
+      <div class="row-hero__image-for-small for-small"><img src="{{ ASSET_SERVER_URL }}3fb3f433-MAAS-takeover-illustration.svg" alt="MAAS service blocks illustration" /></div>
+      <p><a href="https://pages.ubuntu.com/eBook-MAAS.html?utm_source=ubuntu.com&amp;utm_medium=takeover&amp;utm_campaign=MAAS_eBook " onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Maas takeover - 24-05-2016', 'eventLabel' : 'Download the eBook', 'eventValue' : undefined });" class="button--primary">Download the eBook</a></p>
```
Comment (deadlight) [inline templates/takeovers/_maas_may_2017.html]: The event date and name need to be updated here

**Pull request merged**